tests/integration/telemetry/test_telemetry.py

```diff
@@ -50,6 +50,7 @@ def setup_telemetry_data(llama_stack_client, text_model_id):
     yield
 
 
+@pytest.skip(reason="Skipping telemetry tests for now")
```

Comment:
@pytest.mark.skip is the proper format here I think, pytest.skip skips the whole file
```suggestion
@pytest.mark.skip 
```

Reply:
🤦